src/view/stylesmap.js

```diff
@@ -885,3 +885,21 @@ function appendStyleValue( stylesObject, nameOrPath, valueOrObject ) {
  *
  * @typedef {Array.<String, String>} module:engine/view/stylesmap~PropertyDescriptor
  */
+
+/**
+ * An object describing box edges values.
+ *
+ *		const margin = {
+ *			top: '1px',
+ *			right: '3px',
+ *			bottom: '3px',
+ *			left: '7px'
+ *		}
+ *
+ * @typedef {Object} module:engine/view/stylesmap~BoxEdges
```

Comment:
I think it should rather be box **sides** instead. Check out how w3 uses both terms https://www.w3.org/TR/CSS2/box.html (search for both):

### These are for edges:

> The perimeter of each of the four areas (content, padding, border, and margin) is called an "**edge**", so each box has four edges:
> * content **edge** or inner **edge**
> * padding **edge**
> * border **edge**
> ...

![image](https://user-images.githubusercontent.com/1099479/73054139-eb755b80-3e89-11ea-9014-6cc08c68ab65.png)

### These are for **sides** (and IMO this is what we want to say):

> The 'padding' shorthand property sets the padding for all four **sides** while the other padding properties only set their respective side.

> If there is only one component value, it applies to all **sides**. If there are two values, the top and bottom paddings are set to the first value and the right and left paddings are set to the second.

> The 'border-color' property can have from one to four component values, and the values are set on the different sides as for 'border-width'.

### Summary

So **edges** are more like contours **around** the box (containing or not containing the padding, border, etc.) and the box has always 4 **sides** (top, left, bottom, right).

Reply:
Thanks! I've looked at MDN docs and saw the edges used mostly. But your explanaition & wording is better. I'll fix that :)